web/src/SecondaryNavKeyboardShortcuts.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { MemoryRouter, useHistory } from "react-router"
+import React from "react"
+import { createMemoryHistory } from "history"
+import { mount } from "enzyme"
+import SecondaryNavKeyboardShortcuts from "./SecondaryNavKeyboardShortcuts"
+import { fireEvent } from "@testing-library/dom"
+
+var fakeHistory: any
+const shortcuts = (logUrl: string, alertsUrl: string, facetsUrl: string) => {
+  let CaptureHistory = () => {
+    fakeHistory = useHistory()
+    return <span />
+  }
+
+  mount(
+    <MemoryRouter initialEntries={["/"]}>
+      <CaptureHistory />
+      <SecondaryNavKeyboardShortcuts
+        logUrl={logUrl}
+        alertsUrl={alertsUrl}
+        facetsUrl={facetsUrl}
+      />
+    </MemoryRouter>
+  )
+}
+
+it("navigates to logs", () => {
```

Comment:
again, no strong feelings, but would it be useful to test that eg ctrl+1 doesn't get picked up as a nav event?

Reply:
added